### PR TITLE
Rename cost concept to stats

### PR DESCRIPTION
**Pull request by findepi**
1. `PlanNodeCost` that currently exists in master represents statistics (currently: row count, total size in bytes)
2. this commit gives it a more appropriate name (`PlanNodeStatsEstimate`)
3. a class representing cost (true cost) will be introduced later

### CODE REVIEW

**Comment by findepi** [comment]
cc @martint

File: presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java [hunk 1]
```diff
@@ -1289,26 +1289,26 @@ private String formatDomain(Domain domain)
...
         }
 
-        private void printCost(int indent, PlanNode... nodes)
+        private void printPlanNodesStats(int indent, PlanNode... nodes)
         {
-            if (Arrays.stream(nodes).anyMatch(this::isKnownCost)) {
-                String costString = Joiner.on("/").join(Arrays.stream(nodes)
-                        .map(this::formatCost)
+            if (Arrays.stream(nodes).anyMatch(this::isPlanNodeStatsKnown)) {
+                String formatedStats = Joiner.on("/").join(Arrays.stream(nodes)
```
Comment (rschlussel2) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java]: nit: should be formattedStats (double 't' in formatted)

**Comment by martint** [comment]
Can you explain why?

Cost and stats are two different concepts. Stats describe properties of a given plan such as cardinalities, distributions, unique counts, etc. Cost is a metric (possibly multivariate) of how suitable a given plan is.

**Comment by rschlussel2** [comment]
@martint if you look at the fields of PlanNodeStatsEstimate, they relate to stats and not cost (row counts, size of the data), so it was misnamed in the first place.

**Comment by findepi** [comment]
@martint , as @rschlussel2 said:
1. `PlanNodeCost` that currently exists in master represents statistics (currently: row count, total size in bytes)
2. this commit gives it a more appropriate name (`PlanNodeStatsEstimate`)
3. a class representing cost (true cost) will be introduced later

**Pull request merged**